### ISSUE TITLE
Missing dependency on ros-noetic-diagnostic-updater

### ISSUE THREAD

**Issue by WysokiStudent** [ros-noetic-arch/ros-noetic-amcl#1]
I think the PKGBUILD is missing a dependency on `ros-noetic-diagnostic-updater`. If the aforementioned package is not installed, the configure stage fails with:
```
-- Could NOT find diagnostic_updater (missing: diagnostic_updater_DIR)
-- Could not find the required component 'diagnostic_updater'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.
CMake Error at /opt/ros/noetic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
  Could not find a package configuration file provided by
  "diagnostic_updater" with any of the following names:

    diagnostic_updaterConfig.cmake
    diagnostic_updater-config.cmake
```

**Comment by acxz** [comment]
Thanks for the report!
It seems like we do need the dep:
https://github.com/ros-planning/navigation/blob/db50577971b3489e2a693820b53aae198400c443/amcl/package.xml#L31

**Comment by acxz** [comment]
FIxed with 429de75
